_maps/RandomZLevels/caves.dmm

```diff
@@ -489,7 +489,7 @@
 	desc = "An old rune inscribed on the floor, giving off an intense amount of heat.";
 	name = "flame rune"
 	},
-/turf/open/floor/engine/cult{
+/turf/open/floor/plasteel/cult{
```

Comment:
this is not the correct path; change it back, as it will generate an appearance on map load.